airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -7,6 +7,7 @@
     "title": "S3 Destination Spec",
     "type": "object",
     "required": [
+      "s3_endpoint",
```

Comment:
This field should be optinoal. When empty, it can default to "aws". Otherwise, existing users will have to run a migration, which is unnecessary.